utils/gyb_sourcekit_support/UIDs.py

```diff
@@ -182,6 +182,7 @@ def __init__(self, internal_name, external_name):
     KEY('CompletionCheckDependencyInterval',
         'key.completion_check_dependency_interval'),
     KEY('AnnotatedTypename', 'key.annotated.typename'),
+    KEY('CompileOperation', 'key.compileoperation'),
```

Comment:
We're a bit inconsistent about this, but I would suggest `key.compile_operation`

Reply:
Done